api/filter_dimensions.go

```diff
@@ -56,7 +57,8 @@ func (api *FilterAPI) getFilterBlueprintDimensionsHandler(w http.ResponseWriter,
 		return
 	}
 
-	b, err := json.Marshal(filter.Dimensions)
+	publicDimensions := createPublicDimensions(filter.Dimensions)
+	bytes, err := json.Marshal(publicDimensions)
```

Comment:
Stick with `b` over `bytes` as variable `bytes` will conflict with the bytes package